main.go

```diff
@@ -75,9 +74,9 @@ BE ON THE LOOKOUT !
 		}
 
 		input = ""
+		fmt.Println("Players final score is :", score)
```

Comment:
I'm not clear why this line was moved. Because it to show the final score it should happen when we break out of the loop, not after input. I think it was in the right place before.